magda-authorization-api/src/createApiRouter.ts

```diff
@@ -226,6 +228,164 @@ export default function createApiRouter(options: ApiRouterOptions) {
         });
     });
 
+    router.get(
+        "/public/orgUnits/getRootNode",
```

Comment:
So sorry I didn't see this the first time - but I think the fact that it's an HTTP `GET` is already GET enough... and we don't have any other upper-case characters in paths anywhere else in the API. Could we make this `GET /public/orgunits/rootnode`? And could we do that for the other paths in this API too? Hope that's ok 😬

Reply:
- All GET API path updated --- let me know if they are OK.
- The org unit's data has also been integrated to `whoami` api and OPA env.
You can check it with:
https://issue-2185.dev.magda.io/api/v0/opa/data/system/echo
- Also changed the `insertNode` parameters order (feel it's more natural to put `parentNodeId` in front of `newNode` 😄 ). It's now: 
1. `parentNodeId`
2. `newNode`
org-units